src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1003,16 +1003,16 @@ public function get_errors( $args = array() ) {
 	 */
 	public function log_error( WP_Error $error ) {
 		$today = getdate();
-		$args = array(
-			'number' => 1,
-			'date_query' => array(
-				array(
+		$args = [
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space

<sub>hash: 54e50a228df78e19e52151a42c222cd7</sub>